Modules/_testcapimodule.c

```diff
@@ -2515,6 +2515,108 @@ code_offset_to_line(PyObject* self, PyObject* const* args, Py_ssize_t nargsf)
 }
 
 
+typedef struct {
```

Comment:
I'd like to keep the test simple. Here's an example of what I mean:

https://gist.github.com/colesbury/7a7c8d5d3fe54b88532dbf6948e7af70